examples/pytorch/model_zoo/chem/generative_models/jtnn/README.md

```diff
@@ -0,0 +1,103 @@
+# Junction Tree Variational Autoencoder for Molecular Graph Generation (JTNN)
+
+Wengong Jin, Regina Barzilay, Tommi Jaakkola. 
+Junction Tree Variational Autoencoder for Molecular Graph Generation. 
+*arXiv preprint arXiv:1802.04364*, 2018.
+
+JTNN uses algorithm called junction tree algorithm to form a tree from the molecule graph. 
+Then the model will encode the tree and graph into two separate vectors `z_G` and `z_T`. Details can
+be found in original paper. The brief process is as below (from original paper):
+![image](https://user-images.githubusercontent.com/8686776/63677300-3fb6d980-c81f-11e9-8a65-57c8b03aaf52.png)
```

Comment:
Seems that `below (from original paper)` is now below the figure in github after [rendering](https://github.com/VoVAllen/dgl/tree/jtnn_model_zoo/examples/pytorch/model_zoo/chem/generative_models/jtnn). Try moving the image one line below?